index.php

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1">
+    <title>Camera Collection App</title>
+    <link rel="stylesheet" href="collectionApp.css">
+</head>
+<body>
+
+<h1>Camera Collection App</h1>
+
+<div class="mainContainer">
+    <?php
+    require_once 'dbCameras.php';
+    global $db;
+    $query = $db->prepare("SELECT `brandName`, `model`, `type`, `megapixels`, `weight(g)`, `images` FROM `cameras`");
+    $query->execute();
+    $results = $query->fetchAll();
+    
+    foreach ($results as $result) {
+        echo '<div class="imageAndTextContainer">';
+        echo '<div class = "imageContainer">';
+        echo '<img class="imagesClass" src="' . $result['images'] . '" alt="camera">';
+        echo '<br><br>';
```

Comment:
Rather than using `<br>` tags, it's usually better to add padding/margin underneath the image tag